packages/roosterjs-editor-plugins/lib/plugins/ContentEdit/features/listFeatures.ts

```diff
@@ -199,6 +223,17 @@ function getListChains(editor: IEditor) {
     return VListChain.createListChains(editor.getSelectedRegions());
 }
 
+function getCacheNextSibiling(event: PluginKeyboardEvent, editor: IEditor) {
+    const element = cacheGetEventData(event, 'nextSibiling', () => {
+        const range = editor.getSelectionRange();
+        const pos = Position.getEnd(range).normalize();
+        const traverser = editor.getBodyTraverser(pos.node);
+        const blockElement = traverser.getNextBlockElement().collapseToSingleElement();
```

Comment:
I suggest we don't call collapseToSingleElement here because that may change the dom structure.

Instead, you can just use getStartNode(), then find if it is LI node or contained by LI node. You can call findClosestElementAncestor(startNode, 'li') to check.

Reply:
I'll fix!